groups/alteryx-sandbox/locals.tf

```diff
@@ -2,20 +2,20 @@
 # Locals 
 # ------------------------------------------------------------------------
 locals {
-  internal_cidrs               = values(data.vault_generic_secret.internal_cidrs.data)
-  alteryx_key                  = local.secrets.public-key
-  azure_dc_cidrs               = jsondecode(local.secrets.azure_dc_cidrs)
-  concourse_cidrs              = local.automation_subnet_cidrs
-  ansible_cidr_blocks          = join(",", "${local.internal_cidrs}","${local.concourse_cidrs}")
-  account_ids_secrets          = jsondecode(data.vault_generic_secret.account_ids.data_json)
-  vpc_name                     = local.secrets.vpc_name
-  automation_subnet            = local.secrets.automation_subnets_pattern
-  alteryx_subnets_pattern      = local.secrets.alteryx_subnets_pattern
-  alteryx_server_ami_id        = var.alteryx_server_ami_id == "" ? data.aws_ami.alteryx_server_ami[0].id : var.alteryx_server_ami_id
-  alteryx_server_ami_owner_id  = local.account_ids_secrets["shared-services"]
-  secrets                      = data.vault_generic_secret.secrets.data
-  alteryx_worker_ami_id        = var.alteryx_worker_ami_id == "" ? data.aws_ami.alteryx_worker_ami[0].id : var.alteryx_worker_ami_id
-  alteryx_worker_ami_owner_id  = local.account_ids_secrets["shared-services"]
+  internal_cidrs              = values(data.vault_generic_secret.internal_cidrs.data)
+  alteryx_key                 = local.secrets.public-key
+  azure_dc_cidrs              = jsondecode(local.secrets.azure_dc_cidrs)
+  concourse_cidrs             = local.automation_subnet_cidrs
+  ansible_cidr_blocks         = join(",", "${local.internal_cidrs}", "${local.concourse_cidrs}")
```

Comment:
We should move away from concatenated lists of CIDRs and utilise prefix lists instead, where appropriate